using_openshift/managing_nodes.adoc

```diff
@@ -0,0 +1,128 @@
+= Managing Nodes
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+As an OpenShift administrator, you can manage link:../architecture/kubernetes_infrastructure.html#node[nodes] in your instance using the CLI.
+
+The CLI manages and provides information about known nodes in your instance by interacting with node objects. A node object is a representation of a node, which can be a physical or virtual system. The master uses this information to validate the node with health checks. See the https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/node.md#node-management[Kubernetes documentation] for more information on node management.
+
+== Listing Nodes
+To list all nodes that are known to your OpenShift instance, run `osc get nodes`:
+
+----
+$ osc get nodes
+NAME                 LABELS              STATUS
+node1.example.com    <none>              Ready
+node2.example.com    <none>              NotReady
+----
+
+To list information about a single node, run `osc get node [replaceable]#Node_Name#`. For example:
+
+----
+$ osc get node node1.example.com
+NAME                 LABELS              STATUS
+node2.example.com    <none>              NotReady
+----
+
+The `STATUS` column of this output indicates whether the node is passing health checks from the master. Nodes can be shown in this column with the following conditions:
+
+.Node Conditions
+[cols="3a,8a",options="header"]
+|===
+
+|Condition |Description
+
+|`Ready`
+|The node has returned `StatusOK` for the health check.
+
+|`NotReady`
+|The health check is not passing.
+|===
+
+NOTE: The `STATUS` column can also show `Unknown` for a node if the CLI cannot find any node condition.
+
+You can also get more detailed information, including the reason for the current condition, by using the `osc describe node` command for a given [replaceable]#Node_Name#:
+
+[options="nowrap"]
+----
+$ osc describe node node2.example.com
+Name:	node2.example.com
+Conditions:
+Kind    Status  LastProbeTime                   LastTransitionTime              Reason                                                              Message
+Ready   None    Tue, 03 Mar 2015 22:53:22 +0000 Tue, 03 Mar 2015 21:27:50 +0000 Node health check failed: kubelet /healthz endpoint returns not ok
+----
+
+== Adding a Node
+To add a node to an existing OpenShift instance:
+
+. Create a file with the following JSON format:
++
+.[filename]#node.json# file
+====
+
+----
+{
+  "id": "node3.example.com", <1>
+  "kind": "Node", <2>
+  "apiVersion": "v1beta1", <3>
+  "resources": { <4>
```

Comment:
Remove 'resources' field. Currently in OpenShit, we don't recommend users to fill 'resources' field in the node manifest. For more details, look at this card: https://trello.com/c/6llj6hoT/369-3-auto-generate-node-resources-before-adding-the-node-to-the-cluster-admin-install